pkg/virtctl/vm/expand_test.go

```diff
@@ -0,0 +1,170 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package vm_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests/clientcmd"
+)
+
+const (
+	fileInput     = "--file"
+	outputFormat  = "--output"
+	nonExistingVM = "non-existing-vm"
+	invalidFormat = "test-format"
+)
+
+var _ = Describe("Expand command", func() {
+	var vm *v1.VirtualMachine
+	var err error
+	var file *os.File
+	var vmInterface *kubecli.MockVirtualMachineInterface
+	var expandSpecInterface *kubecli.MockExpandSpecInterface
+	var ctrl *gomock.Controller
+	vmName := "testvm"
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		kubecli.GetKubevirtClientFromClientConfig = kubecli.GetMockKubevirtClientFromClientConfig
+		kubecli.MockKubevirtClientInstance = kubecli.NewMockKubevirtClient(ctrl)
+		expandSpecInterface = kubecli.NewMockExpandSpecInterface(ctrl)
+		vmInterface = kubecli.NewMockVirtualMachineInterface(ctrl)
+		vm = kubecli.NewMinimalVM(vmName)
+	})
+
+	It("should fail with missing input parameters", func() {
+		cmd := clientcmd.NewRepeatableVirtctlCommand("expand")
+		Expect(cmd()).NotTo(Succeed())
+	})
+
+	It("should succeed when called on vm", func() {
+		kubecli.MockKubevirtClientInstance.EXPECT().VirtualMachine(k8smetav1.NamespaceDefault).Return(vmInterface).Times(1)
+		vmInterface.EXPECT().GetWithExpandedSpec(context.Background(), vmName)
+
+		cmd := clientcmd.NewVirtctlCommand("expand", "--vm", vmName)
```

Comment:
Mixed use of `clientcmd.NewRepeatableVirtctlCommand` and `clientcmd.NewVirtctlCommand`?